api/machines.go

```diff
@@ -27,6 +30,10 @@ import (
 	"github.com/coreos/fleet/schema"
 )
 
+var (
+	metadataPathRegex = regexp.MustCompile("^/([0-9a-f]{32})/metadata/([A-Za-z0-9_.-]+$)")
```

Comment:
Systemd defines a machine id this way.

http://www.freedesktop.org/software/systemd/man/machine-id.html